Recast/SearchExampleViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SearchViewController.swift
+//  Recast
+//
+//  Created by Drew Dunne on 9/15/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class SearchExampleViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        SearchEndpoint(parameters: ["term": "#DORK", "media": "podcast", "limit": 1]).run()
+            .success { (response: SearchResults) in
+                guard response.resultCount > 0,
+                    let partial = response.results.first else { return }
+                Podcast.loadFull(from: partial, success: { podcast in
+                    print(podcast.title!)
+                    print(podcast.description!)
+                    print(podcast.link!)
+                    print(podcast.collectionName ?? "")
+                    print(podcast.items[1].title ?? "")
```

Comment:
is there a safer way to get stuff out of `items` aside from indexing into the array? is it guaranteed to have the 1st index?

Reply:
also, remove these print statements when you merge in

Reply:
Oh this file is just an example so people can see how to use it. Those '!' are actually unnecessary and the items is all optional, so we will want our views to be able to handle these cases